File: C/Exercicio26.c
int main(int argc, char *argv[]){
  int numero, r;
 
  printf("Digite um numero para tabuada: ");
  scanf("%i", &numero);
 
  while(numero <= 0){
    printf("Erro! Apenas numeros positivos. Digite novamente: ");
    scanf("%i", &numero);
  }
 
  for (int i=0; i <= 10; i++){
    r = numero * i;
    printf("%i x %i = %i\n", numero, i, r);
  }
 
  return 0;
}

File: Python/Exercicio26.py
num = int(input("Digite um numero para tabuada: "))

while(num <= 0):
    num = int(input("Erro! O numero digitado é negativo. Insira um valor positivo: "))
 
for i in range(1, 11, 1):
    r = num*i
    print(f"{num} X {i} = {r}")